setup.py

```diff
@@ -36,7 +36,7 @@ def get_version():
     python_requires=">=3.7",
     install_requires=get_requirements(),
     extras_require={
-        "tests": ["pytest", "mmdet", "norfair"],
+        "tests": ["pytest", "mmdet", "norfair", "layer"],
```

Comment:
We should also update the [ci workflow file](https://github.com/mecevit/sahi/blob/main/.github/workflows/ci.yml) by this addition: 

```yaml
      - name: Install YOLOv5(6.1.2) and Norfair(0.4.3)
        run: >
          pip install yolov5==6.1.2 norfair==0.4.3
```

Reply:
Done: https://github.com/obss/sahi/pull/462/commits/cf9ddbe1ebc3b4669307aaa555ee28c9ab95eefb